codalab/model/bundle_model.py

```diff
@@ -2217,12 +2218,32 @@ def update_user_info(self, user_info):
                 cl_user.update().where(cl_user.c.user_id == user_info['user_id']).values(user_info)
             )
 
-    def increment_user_time_used(self, user_id, amount):
```

Comment:
I think we should be checking how 'amount' is calculated, as in meeting we wanted to make sure amount should only be the 'runnning' time of a run bundle